packages/libs/react-components/src/typings/index.ts

```diff
@@ -0,0 +1,6 @@
+import { IThemeBreakpointProps } from '../styles';
+
+export type Primitive = number | string | boolean;
+export type ICSSPropType =
+  | Primitive
+  | Partial<Record<IThemeBreakpointProps, Primitive>>;
```

Comment:
Is there a reason you opted for `color: { "sm": "$blue500", "md": "$gray500"}` instead of `color: { "@sm": "$blue500", "@md": "$gray500"}`?

The reason I ask is because I think the `@` symbol makes it very clear that we are referring to a breakpoint and this is a style `at size sm`. It also would help us reduce the number of types

Reply:
Additionally, I think it would be valuable to make this type more strict by making it a generic so that we can pass in the property we will apply it to. 
For example: backgroundColor 
`ICSSPropType<Stitches.PropertyValue<"backgroundColor">>`